scripts/create-tree.ts

```diff
@@ -1,17 +1,66 @@
 import { program } from "commander";
+import fs from "fs";
+import { utils } from "ethers";
+import * as sdk from "@across-protocol/sdk-v2";
 
-// TODO
-// Get inspired from: https://github.com/UMAprotocol/protocol/blob/master/packages/merkle-distributor/scripts/1_CreateClaimsForWindow.ts
+import { recipientsFileSchema } from "../src/schemas";
+import { checkRecipientAmountsAndDuplicates } from "../src/validation";
+
+const { MerkleDistributor } = sdk.merkleDistributor;
 
 program
   .requiredOption(
     "-i, --input <path>",
-    "input JSON file location containing a recipients payout"
+    "input JSON file location containing recipients"
   )
   .parse(process.argv);
 
 async function main() {
-  console.log("TODO");
+  const options = program.opts();
+
+  console.log(`Creating a merkle tree and proofs for ${options.input}...`);
+
+  if (!fs.existsSync(options.input)) {
+    throw new Error(`File ${options.input} does not exist`);
+  }
+
+  const inputFile = JSON.parse(
+    fs.readFileSync(options.input, { encoding: "utf8" })
+  );
+
+  console.log("\n1. Validating input file...");
+  const validInputFile = recipientsFileSchema.validateSync(inputFile, {
+    abortEarly: false
+  });
+
+  console.log("\n2. Checking recipient amounts and duplicates...");
+  checkRecipientAmountsAndDuplicates(
+    validInputFile.recipients,
+    validInputFile.rewardsToDeposit
+  );
+
+  console.log("\n3. Creating tree...");
+  const { recipientsWithProofs, merkleRoot } =
+    MerkleDistributor.createMerkleDistributionProofs(
+      validInputFile.recipients.map((r, i) => ({
+        ...r,
+        accountIndex: i,
+        account: utils.getAddress(r.account)
+      })),
+      validInputFile.windowIndex
+    );
+
+  const outputFilePath = `${process.cwd()}/output.json`;
+  const outputFile = {
+    chainId: validInputFile.chainId,
+    rewardToken: validInputFile.rewardToken,
+    windowIndex: validInputFile.windowIndex,
+    rewardsToDeposit: validInputFile.rewardsToDeposit,
+    merkleRoot,
+    recipientsWithProofs
+  };
+  fs.writeFileSync(outputFilePath, JSON.stringify(outputFile, null, 2));
```

Comment:
I think we should make filenames unique, so we don't have old files overwritten. Maybe we can include the date and the window index in the filename. It's also possible that the generated files will be committed on GitHub too